mem0/llms/openai.py

```diff
@@ -18,11 +18,11 @@ def __init__(self, config: Optional[BaseLlmConfig] = None):
         if os.environ.get("OPENROUTER_API_KEY"):  # Use OpenRouter
             self.client = OpenAI(
                 api_key=os.environ.get("OPENROUTER_API_KEY"),
-                base_url=self.config.openrouter_base_url,
+                base_url=self.config.openrouter_base_url or "https://openrouter.ai/api/v1",
```

Comment:
You should either add `OPENROUTER_API_BASE` or it's good to keep `"https://openrouter.ai/api/v1"` as default in config.

Reply:
> keep "https://openrouter.ai/api/v1" as default in config

I think this method still with the same issue: when we set `OPENROUTER_API_BASE` env variable, which won't be used because `base_url=self.config.openrouter_base_url` will always set the `base_url` with `"https://openrouter.ai/api/v1"`

Reply:
Yes makes sense. Thanks